documentation/asciidoc/computers/config_txt/note_moved_bookworm.adoc

```diff
@@ -0,0 +1 @@
+NOTE: Prior to version 12 (_Bookworm_), Raspberry Pi OS stored the boot configuration in a partition named `/boot/`. Versions 12 renamed the boot partition to `/boot/firmware/`.
```

Comment:
In Raspberry Pi OS, we don't really ever refer to the version-numbers (11, 12), we only mention the codenames (Bullseye, Bookworm).